harness/harness.go

```diff
@@ -47,10 +47,12 @@ type Harness struct {
 	proxy      *httputil.ReverseProxy
 }
 
-func renderError(w http.ResponseWriter, r *http.Request, err error) {
-	req, resp := revel.NewRequest(r), revel.NewResponse(w)
-	c := revel.NewController(req, resp)
-	c.RenderError(err).Apply(req, resp)
+func renderError(iw http.ResponseWriter, ir *http.Request, err error) {
+	context := revel.NewGOContext(nil)
```

Comment:
This code need to run a specific engine to display any compile errors, if the compile succeeds then the application is launched and this turns into a proxy